welcome/index.adoc

```diff
@@ -32,7 +32,7 @@ a|link:../admin_guide/index.html[*Administrator Guide*]
 
 These topics describe how to use OpenShift as an administrator.
 
-a|link:../dev_guide/index.html[*Developer Guide*]
+a|link:../registry_quickstart/developers.html[*Developer Guide*]
```

Comment:
Was this intentional? This is updating the welcome page for Origin to point to the registry developer quickstart. But it looks like you're taking care of the "welcome" material for AR in `index-atomic-registry.html`.